apps/dashboard/src/components/activity/components/status-badge.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+import { Badge, BadgeVariant } from '@/components/primitives/badge';
+import { Popover, PopoverContent, PopoverTrigger } from '@/components/primitives/popover';
+import { IActivityJob, JobStatusEnum } from '@novu/shared';
+import { StatusPreviewCard } from './status-preview-card';
+import { JOB_STATUS_CONFIG } from '../constants';
+
+export interface StatusBadgeProps {
+  jobs: IActivityJob[];
+}
+
+export function StatusBadge({ jobs }: StatusBadgeProps) {
+  const [isOpen, setIsOpen] = useState(false);
+  const errorCount = jobs.filter((job) => job.status === JobStatusEnum.FAILED).length;
+  let hoverTimeout: NodeJS.Timeout;
+
+  const handleMouseEnter = () => {
+    clearTimeout(hoverTimeout);
+
+    hoverTimeout = setTimeout(() => {
+      setIsOpen(true);
+    }, 300);
+  };
+
+  const handleMouseLeave = () => {
+    clearTimeout(hoverTimeout);
+    setIsOpen(false);
+  };
+
+  const status = getActivityStatus(jobs);
+
+  const { variant, icon: Icon, label } = JOB_STATUS_CONFIG[status] || JOB_STATUS_CONFIG[JobStatusEnum.PENDING];
+  const displayLabel =
+    status === JobStatusEnum.FAILED ? `${errorCount} ${errorCount === 1 ? 'ERROR' : 'ERRORS'}` : label;
+
+  return (
+    <Popover open={isOpen}>
+      <PopoverTrigger onMouseEnter={handleMouseEnter} onMouseLeave={handleMouseLeave}>
+        <Badge variant={variant as BadgeVariant} className="cursor-pointer gap-1.5">
+          <Icon className="h-3.5 w-3.5" />
```

Comment:
Why are we not using the same icons as what we have in the workflows list page for the success and error? The UI components should be everywhere consistent.